catalog/transformer/catalog-transformer-streaming-lib/src/main/java/org/codice/ddf/transformer/xml/streaming/lib/MetacardTypeRegister.java

```diff
@@ -59,7 +59,7 @@ public void unbind(SaxEventHandlerFactory saxEventHandlerFactory) {
     public void setMetacardType() {
```

Comment:
This doesn't need to be public that I can see. It'd be better as a private method if that's possible.